cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java

```diff
@@ -1522,6 +1532,44 @@ private TSStatus processNonPartitionedDataPlan(PhysicalPlan plan) {
     }
   }
 
+  DeleteTimeSeriesPlan getDeleteTimeseriesPlanWithFullPaths(DeleteTimeSeriesPlan plan)
+      throws PathNotExistException {
+    List<Path> originalPaths = plan.getPaths();
+    ConcurrentSkipListSet<Path> fullPaths = new ConcurrentSkipListSet<>();
+    ConcurrentSkipListSet<String> nonExistPaths = new ConcurrentSkipListSet<>();
+    ExecutorService getAllPathsService = Executors
```

Comment:
Suggestion: change the method signature to `List<String> getMatchedPaths(List<Path> originalPaths)` so that this method can be applied in other situations.

Reply:
Good idea! I will extract the method.